app/views/standup_meeting_groups/joins/create.turbo_stream.erb

```diff
@@ -0,0 +1,4 @@
+<%= turbo_stream.remove @standup_meeting_group %>
+<%= turbo_stream.prepend 'my-standup-meeting-groups' do %>
+  <%= render StandupMeetingGroup::JoinedCardComponent.new(standup_meeting_group: @standup_meeting_group, current_user: current_user) %>
+<% end %>
```

Comment:
So this comes out looking like this after the admin creates a new standup:
![Selection_226](https://github.com/agency-of-learning/PairApp/assets/88392688/b096bb5f-adee-48a3-911a-aa173dc95d06)

I don't particularly like this *especially* because those 'edit' and 'delete' buttons don't actually work. I think it should just pop a new `JoinableItemComponent` in there. We can pull admin edit/delete controls out to a separate ticket and ask Daniel what his vision is for that. They're not actually a part of the wireframes.

Reply:
I put them initially cause I was gonna add them, but I am totally fine ripping them out and dealing with them later.

Reply:
Great catch @JoshDevHub! Didn't realize those buttons didn't work. I agree with you. We'll remove them